code/src/index.css

```diff
@@ -1,13 +1,282 @@
+  font-size: 25px;
+}
+
+p {
+  font-size: 12px;
+  font-weight: bold;
+  margin: 0;
+}
+
+/* FIRST PAGE/SECTION */
+
+header {
+  text-align: center;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+
+.intro {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.intro p {
+  font-size: 24px;
+  font-weight: normal;
+  width: 80%;
+}
+
+.button {
+  width: 200px;
+  height: 40px;
+  border-radius: 5px;
+  border: none;
+  background-color: #F4976C;
+  color: white;
+  color: #2c3866;
+  font-weight: bold;
+  font-size: 18px;
+  margin: 50px;
+}
+
+/* FORM */
+
+.main-form {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  color: #fbe8a6;
+  margin: auto;
+  align-items: center;
+}
+
+.progress-container {
+  width: 100%;
+  height: auto;
+  display: flex;
+  justify-content: space-around;
+  margin: 30px 0;
+
+}
+.progress-section {
+  width: 30px;
+  height: 30px;
+  background-color: none;
+  border: 1px solid #F4976C;
+  margin: 10px;
+  border-radius: 50%;
+}
+
+.progress-active {
+  background-color: #F4976C;
+}
+
+.question-section {
+  width: 100%;
+  height: 60%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.next-button {
+  width: 200px;
+  height: 40px;
+  border-radius: 5px;
+  border: none;
+  border: 2px solid  #D2FDFF;
+  background-color:#2c3866;
+  color:  #D2FDFF;
+  font-weight: bold;
+  font-size: 18px;
+  margin: 50px auto;
+}
+
+
+label {
+  margin: 10px 20px;
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+}
+
+/* TEXT INPUT */
+
+.text-label {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+}
+
+.text-input {
+  height: 40px;
+  margin: 10px 0;
+  background-color: #303C6C;
+  border: none;
+  border-bottom: 2px solid #D2FDFF;
+  padding: 10px;
+  box-sizing: border-box;
+  width: 100%;
+  color: white;
+  font-size: 18px;
+}
+
+input::placeholder {
+  color: #7780a7;
+}
+
+input:focus {
+  border: none;
+  border: 2px solid #F4976C;
+}
+
+/* RADIO INPUT*/
+
+.radio-label {
+  display: inline-flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 10px 10px;
+}
+
+.radio-description {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 20px;
+}
+.radio-options {
+  display: flex;
+  justify-content: space-between;
+}
+.radio {
+  margin: 40px 20px;
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+}
+
+
+/*SELECT INPUT*/
+
+select {
+  height: 40px;
+  width: 100%;
+  padding: 10px;
+  background-color: #303C6C;
+  border: none;
+  border-bottom: 2px solid #D2FDFF;
+  color: white;
+  font-size: 18px;
+}
+
+/* RESULTS PAGE */
+
+.results {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  color: #fbe8a6;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 40px;
+}
```

Comment:
Maybe try to keep a bit more consistent with the order of the styling for each elements/attributes. For example if you put display, display-direction, align-items and justify-content after each other, you know better that they belong together. And also if you keep the same order when you style, let's say for example that every element/attribute keep this order below, instead of changing it all the time, it will be easier and faster to read:

*display
*font-family
*color
*width
*height
*padding
*margin

☺️